### ISSUE TITLE
REPL: wait() + profile.storeIn() leads to crash

### ISSUE THREAD

**Issue by webmaster128** [iov-one/iov-core#252]
`profile.storeIn(db, "secret passwd")` works in the REPL but when wrapping in wait(), we get the following crash:

```
>> wait(profile.storeIn(db, "secret passwd"))
undefined
>>  1: node::Abort() [node]
 2: node::AsyncWrap::EmitTraceEventAfter() [node]
 3: node::AsyncWrap::MakeCallback(v8::Local<v8::Function>, int, v8::Local<v8::Value>*) [node]
 4: 0x5602e03bef5f [node]
 5: uv__run_timers [/usr/lib/x86_64-linux-gnu/libuv.so.1]
 6: uv_run [/usr/lib/x86_64-linux-gnu/libuv.so.1]
 7: node::Start(uv_loop_s*, int, char const* const*, int, char const* const*) [node]
 8: node::Start(int, char**) [node]
 9: __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
10: _start [node]
[1]    23596 abort (core dumped)  ./node_modules/.bin/iov-cli --debug
```

However, this is not urgent since `storeIn` returns just `Promise<void>`.

**Comment by webmaster128** [comment]
Same for `const profileFromDb = wait(UserProfile.loadFrom(db, "secret passwd"));`, which is a problem.

**Comment by webmaster128** [comment]
Is this non-deterministic?

```
>> wait(profile.storeIn(db, "secret passwd"))
undefined
>> const profileFromDb = wait(UserProfile.loadFrom(db, "secret passwd"));
undefined
>>  1: node::Abort() [node]
 2: node::AsyncWrap::EmitTraceEventAfter() [node]
 3: node::AsyncWrap::MakeCallback(v8::Local<v8::Function>, int, v8::Local<v8::Value>*) [node]
 4: 0x55bea1465f5f [node]
 5: uv__run_timers [/usr/lib/x86_64-linux-gnu/libuv.so.1]
 6: uv_run [/usr/lib/x86_64-linux-gnu/libuv.so.1]
 7: node::Start(uv_loop_s*, int, char const* const*, int, char const* const*) [node]
 8: node::Start(int, char**) [node]
 9: __libc_start_main [/lib/x86_64-linux-gnu/libc.so.6]
10: _start [node]
[1]    32502 abort (core dumped)  ./node_modules/.bin/iov-cli --debug
```

**Comment by webmaster128** [comment]
Can be closed together with #274
